### PR TITLE
Fixes for OFED auto-upgrade flow

### PR DESCRIPTION
**Pull request by almaslennikov**

### CODE REVIEW

File: deployment/network-operator/templates/role.yaml [hunk 1]
```diff
@@ -256,3 +256,15 @@ rules:
       - patch
       - update
       - watch
+  - apiGroups:
```
Comment (e0ne) [inline deployment/network-operator/templates/role.yaml:259]: these files https://github.com/Mellanox/network-operator/blob/master/config/rbac/role.yaml and https://github.com/Mellanox/network-operator/blob/master/deploy/role.yaml should be updated too

**Review by e0ne — CHANGES_REQUESTED** [review]

File: deployment/network-operator/templates/role.yaml (hunk 1 above)
Comment (almaslennikov) [reply deployment/network-operator/templates/role.yaml:259]: Thank you! Updated them too

File: pkg/upgrade/drain_manager_test.go [hunk 2]
```diff
@@ -53,6 +53,37 @@ var _ = Describe("DrainManager tests", func() {
...
+			TimeoutSecond:  1,
+			DeleteEmptyDir: true,
+		}
+		nodeArray := []*corev1.Node{node1, node2, node3}
+		err := drainManager.ScheduleNodesDrain(ctx, &upgrade.DrainConfiguration{Nodes: nodeArray, Spec: drainSpec})
+		Expect(err).To(Succeed())
+
+		time.Sleep(time.Second)
+
+		err = k8sClient.Get(ctx, types.NamespacedName{Name: node1.Name}, node1)
```
Comment (adrianchiris) [inline pkg/upgrade/drain_manager_test.go:77]: can you use a new var for the observed node ? e.g `observedNode` instead of reusing node1 ?

**Review by adrianchiris — CHANGES_REQUESTED** [review]
can you re format commit messages so it has header and body ?

otherwise LGTM

File: pkg/upgrade/drain_manager_test.go (hunk 2 above)
Comment (almaslennikov) [reply pkg/upgrade/drain_manager_test.go]: Thanks, addressed

**Review by ykulazhenkov — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

**Review by e0ne — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**